### PR TITLE
Feature/meta data

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

File: app/lib/constants.js [hunk 1]
```diff
@@ -9,6 +9,17 @@ module.exports = {
       open: 'open',
     },
   },
+  app: {
```
Comment (beseven) [inline app/lib/constants.js]: nice use of constants

File: config/contentSecurityPolicy.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ module.exports = {
       'assets.adobedtm.com',
       '*.demdex.net',
       '*.hotjar.com:*',
+      '*.hotjar.io',
```
Comment (beseven) [inline config/contentSecurityPolicy.js]: can you add this change to the changelog too please?

Comment (st3v3nhunt) [reply config/contentSecurityPolicy.js]: Done

**Comment by st3v3nhunt** [comment]
I've updated the dependencies (again) too.

**Comment by st3v3nhunt** [comment]
Resolves #559

**Pull request merged**